ronin/drive-storage.rst

```diff
@@ -0,0 +1,24 @@
+.. _drive-storage:
+
+Drive Storage
+=======================================
+
+Drive storage or block storage as it's often referred as is the storage attached directly to your :term:`instances<instance>` within Ronin.
+These are most commonly the "Root Drive" however Ronin gives you the option to create your own additional storage to attach and move between instances.
+
+Should you want to know about the nitty gritty, the underlying technology used here is `AWS EBS <https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/AmazonEBS.html>`_.
+
+.. _drive_types:
+
+Drive Types
+---------------------------------------
+
+When creating a new non-root drive, Ronin gives you multiple options for drive types.
+As a general rule of thumb we recommend you select **SSD**, this is due to how AWS provisions drive speed.
+
+The SSD drives will be allocated 125MiB/s and 3000iops as per `gp3 defaults <https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/general-purpose.html#gp3-performance>`_.
+Should your workload require more performance please do get in touch.
```

Comment:
I'd like a guide on using the storage-optimised machines properly please 😸 
Most of the data processing tasks for CURED are bottlenecked heavily by disk I/O so users being able to easily optimise this would save a lot of time

Reply:
We could signpost to tooling that helps people quantify IO perf (on Linux and Windows instances - `iostat` and `iotop` for Linux).

A (mermaid.js) flow chart for diagnosing IO perf issues could be useful.

Reply:
Yes please
I've collected a few [tips](https://docs.google.com/document/d/1y9Y1l5B1Vn0VW6-Ie4HP0QTq_CiH2tAcjUs5tW1v09g/edit?usp=sharing) but I realise it's probably a tricky topic for newcomers to grasp. There's a very wide potential variety of experience in the users - some extremely basic concepts to explain or at least signpost to relevant training